app/helpers/rubygems_helper.rb

```diff
@@ -12,7 +12,9 @@ def link_to_page(text, url)
   end
 
   def link_to_directory
-    ("A".."Z").map { |letter| link_to(letter, rubygems_path(:letter => letter)) }.join
+    ("A".."Z").map do |letter|
+      link_to(letter, rubygems_path(:letter => letter), :class => "gems__nav-link")
```

Comment:
@qrush Opss sorry. I changed the other one in test but forget this one. @sferik merged this PR. Should I send another PR for that ?